subsystem/infra_env_test.go

```diff
@@ -350,9 +353,12 @@ var _ = Describe("Infra_Env", func() {
 		contents := buf.String()
 		Expect(len(contents)).ShouldNot(Equal(0))
 		Expect(contents).To(ContainSubstring("/etc/assisted/network/host0"))
-		Expect(contents).To(ContainSubstring("192.0.2.1/24"))
+		//Expect(contents).To(ContainSubstring("192.0.2.1/24"))
 		Expect(contents).To(ContainSubstring("eth0"))
-	})
+	},
+		Entry("ocp versions greater than/ equal to 4.13", staticnetworkconfig.MinimalVersionForNmstatectl),
```

Comment:
should be updated to 4.14, if we decide not to support this flow for 4.14

Reply:
According to Gris, version 2.2.35 will be released for RHEL 9.2 on October 1. It may take 1-2 weeks to see an OCP release that includes it. Since the mac-identifier has been broken in production for some time, we can continue working with version 4.13 until we receive the OCP versions from Gris. In addition, currently, all versions are affected by the mac-identifier issue anyway. what do you think?

Reply:
We can handle it later on: https://github.com/openshift/assisted-service/pull/6695/files#r1756393055